HELM.md

```diff
@@ -1,7 +1,23 @@
 # Helm
 
 [Helm](https://github.com/kubernetes/helm) is a client CLI used to deploy Kubernetes
-applications. It supports templating of configuration files, but also needs a server
+applications.
+
+## Helm 3
```

Comment:
I wonder if it would be cleaner to have a `HELM.md` and `HELM2.md` instead of branching so many sections of the document. 

The current version of helm _is_ helm 3, we should document that and provide a link to existing instructions for helm 2.  Keeping "legacy" instructions inline like this seems unnecessary.

The default install via brew is helm 3 and the first result in google is the install instructions for helm 3.  The quickstart linke on line 23 (below this) even points to the helm 3 docs.  We could include a "not sure of your version, run `helm version`  to check" statement.

Reply:
It already feels like a lot of indirection having the docs go to HELM.md then to charts/openfaas/README.md, having HELM2.md and HELM.md might be more modular, but it could also be a lot of clicks.

I don't know what the perfect version of this is, but let's start with what's there. Thank you for these suggestions.

Reply:
Bear in mind that most of our users (that I'm aware of) are still using helm 2, as are other parts of our docs, blog posts, tutorials and maybe the workshop. I suspect helm3 will be a more graduated move than a strict lift and shift. (I might be wrong)